docs/source/modin/supported/dataframe_supported.rst

```diff
@@ -78,7 +78,8 @@ Methods
 | ``aggregate``               | P                               | ``margins``, ``observed``,       | See ``agg``                                        |
 |                             |                                 | ``sort``                         |                                                    |
 +-----------------------------+---------------------------------+----------------------------------+----------------------------------------------------+
-| ``align``                   | N                               |                                  |                                                    |
```

Comment:
Do we also need to mention `fill_value` here?
Perhaps we can also include the deprecated parameters (`method`, `limit`, `fill_axis`, `broadcast_axis`) if they are not implemented?

Reply:
I added `fill_value` to the missing parameters but now added in Notes too. Sure, adding deprecated params here as well.